app/components/task/holder.js

```diff
@@ -27,6 +28,33 @@ export default class TasksHolderComponent extends Component {
     return statusToDisplay;
   }
 
+  get removedTaskStatus() {
+    const statusToDisplay = this.availabletaskStatusList.filter(
+      (taskStatus) => {
+        if (this.args.dev === true) {
+          if (
+            taskStatus.key !== 'ALL' &&
+            taskStatus.key !== 'VERIFIED' &&
+            taskStatus.key !== 'RELEASED' &&
+            taskStatus.key !== 'REGRESSION_CHECK' &&
+            taskStatus.key !== 'SANITY_CHECK' &&
+            taskStatus.key !== 'APPROVED' &&
+            taskStatus.key !== 'SMOKE_TESTING' &&
+            taskStatus.key !== 'COMPLETED' &&
+            taskStatus.key !== 'NEEDS_REVIEW'
+          ) {
+            return true;
+          } else {
+            return false;
```

Comment:
Re: I think instead of filtering all these statuses. We can use a new constants list